### ISSUE TITLE
How can I visualize datasets?

### ISSUE THREAD

**Issue by Enuang** [ma-xu/pointMLP-pytorch#82]
Thank you for your wonderful work! I would like to see the changes in ScanObjectNN after some data augmentation operations. How can I visualize it?

**Comment by ma-xu** [comment]
@Enuang Thanks for your interests. Mabe you can refer this one and make some modifications to get what you want https://github.com/ma-xu/pointMLP-pytorch/issues/55#issuecomment-1226716347

**Comment by Enuang** [comment]
> @Enuang Thanks for your interests. Mabe you can refer this one and make some modifications to get what you want [#55 (comment)](https://github.com/ma-xu/pointMLP-pytorch/issues/55#issuecomment-1226716347)
Thanks. And how can I visualize the training process?

**Comment by Enuang** [comment]
Thanks. And how can I visualize the training process?

**Comment by ma-xu** [comment]
> Thanks. And how can I visualize the training process?

You can track the training loss of each epoch
